ui/app/components/job-status/deployment-history.hbs

```diff
@@ -33,7 +33,7 @@
       {{#each this.history as |deployment-log|}}
         <li class="timeline-object {{if (eq deployment-log.exitCode 1) "error"}}">
           <div class="boxed-section-head is-light">
-            <LinkTo @route="allocations.allocation" @model={{deployment-log.state.allocation}} class="allocation-reference">{{deployment-log.state.allocation.shortId}}</LinkTo>
+            <LinkTo @route="allocations.allocation" @model={{deployment-log.state.allocation.id}} class="allocation-reference">{{deployment-log.state.allocation.shortId}}</LinkTo>
```

Comment:
If I look at the preview: https://nomad-storybook-and-pufvhadvm-hashicorp.vercel.app/ui/jobs/actively-deploying-job@default there are no deployment history events showing. Is that just a function of not having test data there in the storybook preview?

Reply:
Yeah exactly, it's just that the mocked data there doesn't exist in Mirage (point of clarification: Mirage is the mocked data you see in a deployment, but the dashboard is otherwise the same as you'd see in a real build. Storybook is the components in isolation, like https://nomad-storybook-and-pufvhadvm-hashicorp.vercel.app/storybook/?path=/story/theme-colors--colors)